Documentation/ceph-osd-mgmt.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Ceph OSD Management
+weight: 10610
+indent: true
+---
+
+# Ceph OSD Management
```

Comment:
```suggestion
## Ceph OSD Management
```

Reply:
The top level of all the docs are the first-level titles, so this would be changing the pattern

Reply:
Yes, but markdown lint says something different...

Reply:
I see the error too: `Multiple top level headings in the same document`. It must be confused because of this doc generation section at the top. There is no other actual title.
```
---
title: Ceph OSD Management
weight: 10610
indent: true
---
```